.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: ci
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Similarly, as below should we solely trigger on tags?

```suggestion
```

Reply:
@jadolg here the master branch is still in the PR right? Or am I looking wrong?

Reply:
fixed in [6b72385](https://github.com/Jimdo/yaml-vault/pull/3/commits/6b723853f7c81121db551305d600c7db544d5c88)